src/jsMain/kotlin/ui/Content.kt

```diff
@@ -159,6 +160,7 @@ val Content = FC<AppProps> { props ->
                     }
 
                     val deps = setOf(
+                        DependencyBox(MokoMvvm, false),
```

Comment:
Move it after the KotlinxCoroutinesCore, please (first elements are selected by default)